en/components/time_picker.md

```diff
@@ -237,6 +237,46 @@ And there we have it:
     <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="time-picker-sample-5" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
+#### Templating Input Group - dropdown mode
+
+All the information mentioned in the <a href="#templating-input-group">Templating Input Group</a> section can be applied when re-templating a dropdown mode time picker. The only additional requirement is that an HTML element inside the template should be marked with the `dropDownTarget` template reference variable to be used as an opening target for the dropdown.
```

Comment:
You can add that if no html element has `dropDownTarget` the dropdown will be opened in the center of the parent element.